docs/guides/htpc.md

```diff
@@ -1,95 +1,143 @@
 ---
-label: HTPC (on a budget)
+label: HTPC
 order: -2
-description: Anime on the Big Screen
+description: Learn how to setup a home-theater system for watching anime
 image: /static/tohsaka.gif
 ---
 
-# Why HTPC?
+# HTPC
 
-There are plenty of reasons to want to use a HTPC over a smart TV or other premade TV box. Among them, there is [poor subtitle rendering](https://cdn.discordapp.com/attachments/930734323353677854/1137931348510785586/Kodi_20230806_190720_Record.mp4), [poor software support](https://github.com/mpv-android/mpv-android/issues/12), or limited filtering and upscaling options.
+A home-theater PC (HTPC) is a system specifically designed to host/stream media, such as anime TV shows and movies.
 
-As of the writing of this guide, any less than $50 leaves you with the Amazon Fire Stick 4K Max as probably the best option, as most PCs cheaper than this will not handle things much better, and in some cases may be even worse.
+There are plenty of reasons you may want to consider using a HTPC over a smart TV or other premade TV box, such as [bad subtitle rendering](/static/playback/ass-subtitles-issue.mp4), [poor software support](https://github.com/mpv-android/mpv-android/issues/12), or limited filtering and upscaling options.
 
-## Specs Rundown
+!!!
+If you are on a strict budget ($50 or less), we suggest getting an [Amazon Fire TV Stick 4K Max](https://www.amazon.com/dp/B08MQZXN1X), as most PCs at this price will provide a suboptimal experience.
+!!!
+
+## Specifications
+
+There are two main things to consider when building an anime HTPC: the [GPU](#gpu) and the [CPU](#cpu). The information below will help you decide on what to choose for your setup and budget.
+
+### GPU
 
-There are two main considerations for an Anime HTPC. The biggest, will be the CPU. You will need to decide if you want to use the integrated GPU (iGPU) or if you want to install an external dedicated GPU (dGPU).
+Generally, a dedicated graphics card (dGPU) will almost always be better than the integrated option (iGPU). *However, you may want to consider whether your home-theater setup will necessitate one if you're on a tighter budget:*
 
-+++ dGPU
++++ Dedicated
+
+A dGPU is a dedicated piece of hardware which will be used to handle video decoding. Cards in this category are usually from *AMD* or *NVIDIA*, with *Intel* potentially for newer codecs.
```

Comment:
> with *Intel* potentially for newer codecs.
Intel's only advantage was having AV1, but both nvidia and amd have it now with much better drivers than what intel offers. Intel is also new which means it's less bang for your bucks.

Reply:
Alright. Should we add a note instead since Intel makes dGPUs?

Reply:
They already killed their ARC line of GPUs, and currently only have LE (low performance) model. There's no gurantee if they will kill it or not so I think we can just ignore them for now

Reply:
Resolved in https://github.com/Snaacky/thewiki/pull/166/commits/833bbf18904ae7deee475422b3b6489cd05a32a9.